clients/src/main/java/org/apache/kafka/server/telemetry/ClientTelemetry.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.telemetry;
+
+import org.apache.kafka.common.metrics.MetricsContext;
+import org.apache.kafka.common.metrics.MetricsReporter;
+import org.apache.kafka.common.annotation.InterfaceStability;
+
+/**
+ * A {@link MetricsReporter} may implement this interface to indicate support for collecting client
+ * telemetry on the server side.
+ */
+@InterfaceStability.Evolving
+public interface ClientTelemetry {
+
+    /**
+     * Implemented by the broker {@link MetricsReporter} to provide a {@link ClientTelemetryReceiver}
```

Comment:
> Implemented by the broker {@link MetricsReporter} 

Sound weird -- I would not use the term "Implemented" for a method, because it's clear that if one implement the interface they must implement all methods... -- Also "by the broker" sound like as if the was a default implementation what is not the case.

In the end, we should only describe what this method does (might also be good to clarify if it's expected to return a new object on each call; guess we can omit it if there is no restriction and a cached or singleton object could be returned, too):
```
Creates a {@link ClientTelemetryReceiver} instance.
```

Reply:
maybe it would be clearer to say that the broker calls this method to get the ClientTelemetryReceiver instance.

Reply:
Done.